chart/templates/_helpers.tpl

```diff
@@ -146,6 +146,17 @@ imagePullSecrets:
       key: hashsalt
 - name: DRUPAL_CONFIG_PATH
   value: {{ .Values.php.drupalConfigPath }}
+{{- if .Values.php.newrelic.enabled }}
```

Comment:
If we only need a license, and the default license is empty, can we check for the presence of a license instead of a dedicated `enabled` parameter?

Reply:
From existence of license key, infer whether newrelic is required. This is an elegant suggestion, Ill add it.